aries-site/src/examples/foundation/color/ColorPalettes.js

```diff
@@ -0,0 +1,227 @@
+import { useContext } from 'react';
+import { Box, ResponsiveContext } from 'grommet';
+import { ColorRow, UsageExample } from '../../../layouts';
+
+import { colorExamples } from '../../../data';
+import { ElevationExample, TextExample } from '../..';
+
+const {
+  coreColors,
+  darkColors,
+  lightColors,
+  primaryColors,
+} = colorExamples.palettes;
+const {
+  contrastDark,
+  contrastLight,
+  darkBackgrounds,
+  lightBackgrounds,
+} = colorExamples.backgrounds;
+const { borderDark, borderLight } = colorExamples.borders;
+const { inputDark, inputLight } = colorExamples.input;
+const { elevationColorsDark, elevationColorsLight } = colorExamples.elevation;
+const { focusColor, layerColor, graphColor } = colorExamples;
+const { statusColorsDark, statusColorsLight, textColors } = colorExamples.text;
+
+const generateColorExamples = (colors, textColor, textSize) => (
+  <Box fill>
+    {colors.map(color => (
+      <ColorRow
+        colorSpec={color}
+        key={color.name}
+        textColor={textColor}
+        textSize={textSize}
+      />
+    ))}
+  </Box>
+);
+
+export const BrandColor = () =>
+  primaryColors && generateColorExamples(primaryColors, 'text-strong', 'large');
```

Comment:
any reason in particular to make this larger?

Reply:
Contrast, thus readability is poor at "small" font size. Was waffling whether we should leave alone or change.